SpatialGDK/Source/SpatialGDK/Public/Utils/EngineVersionCheck.h

```diff
@@ -7,7 +7,7 @@
 // GDK Version to be updated with SPATIAL_ENGINE_VERSION
 // when breaking changes are made to the engine that requires
 // changes to the GDK to remain compatible
-#define SPATIAL_GDK_VERSION 12
+#define SPATIAL_GDK_VERSION 13
```

Comment:
This will break 423 and 422. We could scope this inside >= 424 #def, or update the older engine branches.

Reply:
good point, will fix